cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractSearchTest.java

```diff
@@ -44,6 +48,7 @@ public static void setUpClass() throws Exception {
     public static void tearDownClass() throws Exception {
         Cloudinary cloudinary = new Cloudinary();
         cloudinary.api().deleteResourcesByTag(SEARCH_TAG, null);
+        cloudinary.api().deleteFolder(SEARCH_FOLDER, null);
```

Comment:
If the folder does not exist will it gracefully exit or crash?

Reply:
Wrapped with try/catch